src/components/ProfilePicture.tsx

```diff
@@ -1,10 +1,24 @@
+interface ProfilePictureProps extends React.ComponentProps<"img"> {
+  imgUrl?: string | null;
+  size: number;
+}
 /**
- * @prop size: Width/Height of the Profile Picture in REM 
- * @returns 
+ * @prop size: Width/Height of the Profile Picture in REM
+ * @returns
  */
-export const ProfilePicture: React.FC<{ imgUrl?: string | null, size: number }> = ({ imgUrl, size }) => {
-    return (
-        <img src={imgUrl || "https://i.stack.imgur.com/34AD2.jpg"} alt="avatar" className="aspect-square rounded-full inline-block" style={{ width: `${size}rem` }} />
-
-    )
-}
\ No newline at end of file
+export const ProfilePicture: React.FC<ProfilePictureProps> = ({
+  imgUrl,
+  size,
+  ...props
+}) => {
+  const { className, ...restProps } = props;
+  return (
+    <img
+      src={imgUrl || "https://i.stack.imgur.com/34AD2.jpg"}
+      alt="avatar"
+      className={`inline-block aspect-square rounded-[50%] ${className ?? ""}`}
+      style={{ width: `${size}rem`, height: `${size}rem` }}
+      {...restProps}
```

Comment:
Had to change this because sometimes the profile pictures were not keeping their circular shape. Specifically, changing `rounded-full` to `rounded-[50%]` and adding the `height` prop to the style object solved the problem.